README.md

```diff
@@ -158,12 +198,18 @@ The package tests are checked with [Infection](https://infection.github.io/) mut
 
 ### Static analysis
 
-The code is statically analyzed with [Phan](https://github.com/phan/phan/wiki). To run static analysis:
+The code is statically analyzed with [Psalm](https://psalm.dev/). To run static analysis:
```

Comment:
This package still uses phan static analysis.

Reply:
Yes, phan. Why you change in readme phan to psalm ?

Reply:
I make another PR (https://github.com/yiisoft/event-dispatcher/pull/42) which change phan to psalm.

Reply:
Maybe this naming will be better 
`recordEvent` => `pushEvent`
`releaseEvents` => `getEvents`

Reply:
> `recordEvent` => `pushEvent`
`push` - sending somewhere, but here save to yourself

> `releaseEvents` => `getEvents`

`get` - only get, but here get and clear